lib/absinthe/phase/document/execution/resolution.ex

```diff
@@ -75,11 +75,35 @@ defmodule Absinthe.Phase.Document.Execution.Resolution do
   end
 
   defp run_callbacks(plugins, callback, acc, true) do
-    Enum.reduce(plugins, acc, &apply(&1, callback, [&2]))
+    Enum.reduce(plugins, acc, fn plugin, acc ->
+      start_time = System.monotonic_time()
+
+      emit_start_event(start_time, acc, callback, plugin)
+      acc = apply(plugin, callback, [acc])
+      emit_stop_event(start_time, acc, callback, plugin)
+
+      acc
+    end)
   end
 
   defp run_callbacks(_, _, acc, _), do: acc
 
+  defp emit_start_event(start_time, acc, callback, plugin) do
+    :telemetry.execute(
+      [:absinthe, :plugin, callback, :start],
+      %{start_time: start_time},
```

Comment:
For this to be a useful measurement, it needs to be a `system_time()`... `monotonic_time` is still needed for duration tracking, but it doesn't actually mark a point in time

Reply:
Updated the PR to have the `start_time` be from system_time()